webapp/routes.py

```diff
@@ -92,8 +92,26 @@ def create():
     if flask.request.method == "POST":
```

Comment:
Just curious why would we use both `POST` and `GET` method

Reply:
I was also wondering this. It doesn't seem to have a use case in this route.